hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -117,7 +122,20 @@ export const UnconnectedCheckedOutTable = ({ items, isVisible, toggleVisibility
                                         </IconButton>
                                     </TableCell>
                                     <TableCell align="right">{row.qty}</TableCell>
-                                    <TableCell className={styles.widthBuffer} />
+                                    <TableCell
+                                        align="right"
+                                        className={styles.widthButton}
+                                    >
+                                        <Button
+                                            color="secondary"
```

Comment:
Since this is supposed to be red, will this be resolved by #73?

Reply:
yup, should be red now